### PR TITLE
coala_web.md: Update project description and milestones

### PR DESCRIPTION
**Pull request by hemangsk**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: data/projects/coala_web.md [hunk 1]
```diff
@@ -1,54 +1,59 @@
...
-up coala-html requires `coala-html -h`.
-If coala-html is converted to a cross platform desktop application using
-frameworks such as Electron, we can have a CLI independent tool.
-
-The Sublime Text package control page could be used as inspiration for a
-bear page: <https://packagecontrol.io/>
+This project aims to increase the user base of coala and provide a web based 
+interface to use coala on their projects. The project is conceptually built upon
+ various previous projects (coala-gui, coala-html) which have helped us refine, 
+ redefine and revisit our GUI approach.
```
Comment (sils) [inline data/projects/coala_web.md]: don't do spaces before the words please

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/303/commits/000e3ad5e70e92de301eedc498ee13381e0cc994

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/303/commits/a19bd99d843778379ecdb7d5150e843216536177

**Comment by hemangsk** [comment]
reack 58aba7c

**Comment by sils** [comment]
ack c7991666d2a2af4a97234e303d632ab9032df65c

**Pull request merged**